funnel/templates/admin.html.jinja2

```diff
@@ -25,12 +25,12 @@
         <ol class="mui-list--aligned">
           {%- for event in events %}
             <li>
-              <a href="{{ url_for('event', profile=project.profile.name, project=project.name, name=event.name) }}" class="mui--text-subhead">{{ event.title }}</a>{%- if project.current_roles.admin %}&nbsp;<a href="{{ url_for('event_edit', profile=project.profile.name, project=project.name, name=event.name) }}" aria-label="{% trans %}Edit{% endtrans %}" title="Edit"><i class="material-icons mui--text-subhead pull-right" aria-hidden="true">edit</i></a>{%- endif %}
+              <a href="{{ url_for('event', profile=project.profile.name, project=project.name, name=event.name) }}" class="mui--text-subhead">{{ event.title }}</a>{%- if project.current_roles.profile_admin %}&nbsp;<a href="{{ url_for('event_edit', profile=project.profile.name, project=project.name, name=event.name) }}" aria-label="{% trans %}Edit{% endtrans %}" title="Edit"><i class="material-icons mui--text-subhead pull-right" aria-hidden="true">edit</i></a>{%- endif %}
             </li>
           {%- endfor %}
         </ol>
       </div>
-      {%- if project.current_roles.admin %}
+      {%- if project.current_roles.profile_admin %}
```

Comment:
`profile_admin` will _not_ be available in the project's roles, and also cannot be used for anything other than editing crew memberships.

Reply:
This has to be changed.